.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+## How the feature works? / How did you fix the issue?
+
+
+## Screenshots, video, GIF if applicable
```

Comment:
Good job on mentioning screenshots, videos, and GIFs! However, usually, stuff like screenshots is mixed  in the answer to _How the feature works_, so I'd suggest merging the two titles into one:
```suggestion
## How does the feature work? / How did you fix the issue?
(Screenshots, videos, or GIFs, if applicable)
```